webapp/channels/src/utils/performance_telemetry/reporter.ts

```diff
@@ -121,6 +132,10 @@ export default class PerformanceReporter {
         // Send any remaining metrics when the page becomes hidden rather than when it's unloaded because that's
         // what's recommended by various sites due to unload handlers being unreliable, particularly on mobile.
         addEventListener('visibilitychange', this.handleVisibilityChange);
+
+        if (!this.desktopAPI.isDev()) {
```

Comment:
Why not collect this for dev builds? I'd expect that collecting this would be helpful for dev builds in some circumstances, but I also think that they wouldn't produce enough output to meaningfully affect results on live servers. As an alternative to not sending any data for dev builds, we could use a `dev` version instead to let us filter those out if we find them problematic

Reply:
`isDev()` denotes that they're running the dev environment. I don't think we need to collect data for that.
We'll still be collecting for `develop` and `nightly` tagged production builds as per above.